### PR TITLE
docs(cli): Clarify CLI-in-docker instructions

### PR DESCRIPTION
**Pull request by bo0tzz**
rel: https://github.com/immich-app/CLI/pull/58

### CODE REVIEW

File: docs/docs/features/bulk-upload.md [hunk 1]
```diff
@@ -46,19 +46,31 @@ The API key can be obtained in the user setting panel on the web interface.
...
-Be aware that as this runs inside a container it mounts your current directory as a volume, and for the -d flag you need to use the path inside the container.
+:::caution Running inside Docker
+Be aware that as this runs inside a container, you need to mount the folder from which you want to import into 
+the container. The below command uses `$(pwd)` to mount the folder you are currently in.
+:::
+
+You can run the CLI inside of a docker container like this:
 
 ```bash
-docker run -it --rm -v $(pwd):/import ghcr.io/immich-app/immich-cli:latest upload --key HFEJ38DNSDUEG --server http://192.168.1.216:2283/api -d /import
```
Comment (alextran1502) [inline docs/docs/features/bulk-upload.md:52]: I think `--d` is a required flag?

Comment (bo0tzz) [reply docs/docs/features/bulk-upload.md:52]: Check the linked CLI PR - I set a default via an env var in the Dockerfile.

**Review by jrasm91 — APPROVED** [review]
Looks GREAT.

File: docs/docs/features/bulk-upload.md [hunk 2]
```diff
@@ -46,18 +46,36 @@ The API key can be obtained in the user setting panel on the web interface.
...
+```
+
+:::tip Internal networking
+If you are running the CLI container on the same machine as your Immich server, you may not be able to reach the external address. In that case, try the following steps:
+1. Find the internal Docker network used by Immich via `docker network ls`.
+2. Adapt the above command to pass the `--network <immich_network>` argument to `docker run`, substituting `<immich_network>` with the result from step 1.
+3. Use `--server http://immich-server/` for the upload command instead of the external address.
+
+```bash title="Upload to internal address"
+docker run --network immich_default -it --rm -v $(pwd):/import ghcr.io/immich-app/immich-cli:latest upload --key HFEJ38DNSDUEG --server http://immich-server/
```
Comment (jrasm91) [inline docs/docs/features/bulk-upload.md:76]: I believe you still need the port here (3001), since we don't run on port 80 internally.

**Review by jrasm91 — APPROVED** [review]
Looks great!

Just missing the port number for the internal example I think.

**Pull request merged**